
alignment_example:
  bag_topic:       /right_wall
  bag_name_1:      /home/conor/ros_data/Final_Thesis_Tests/input_data/1a.bag
  bag_name_2:      /home/conor/ros_data/Final_Thesis_Tests/input_data/1d.bag
#  bag_name_2:      /home/conor/ros_data/trees/18 ahg under bald cypress/laser_stitcher_1538946077.467459.bag